examples/external-data/tests/customerService.test.ts

```diff
@@ -80,7 +130,10 @@ describe("mock-customer-service", () => {
 
 		// Bind listener
 		let wasFluidNotifiedForChange = false;
-		localServiceApp.post("/broadcast-signal", (_, result) => {
+		let webhookChangeNotification;
+		localServiceApp.post("/broadcast-signal", (request, result) => {
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+			webhookChangeNotification = request.body;
```

Comment:
Nit: We can probably remove the eslint rule disable pragma by doing a `!== undefind` check here.